bindings/python/py_src/tokenizers/implementations/sentencepiece_unigram.py

```diff
@@ -49,13 +49,29 @@ def train(
         vocab_size: int = 8000,
         show_progress: bool = True,
         special_tokens: List[Union[str, AddedToken]] = [],
+        unk_token: Optional[str] = None,
     ):
-        """ Train the model using the given files """
+        """
+        Train the model using the given files
+
+        Args:
+            files (:obj:`List[str]`):
+                A list of path to the files that we should use for training
+            vocab_size (:obj:`int`):
+                The size of the final vocabulary, including all tokens and alphabet.
+            show_progress (:obj:`bool`):
+                Whether to show progress bars while training.
+            special_tokens (:obj:`List[Union[str, AddedToken]]`):
+                A list of special tokens the model should know of.
+            unk_token (:obj:`str`, `optional`):
+                The unknown token to be used by the model.
```

Comment:
Not all implementations in the same folder describe method arguments (I think because these objects are not in the documentation). Nevertheless, here I thought it could be really interesting to have them so that the user can understand what the new added `unk_token` argument is for.

Reply:
Yes, totally fine.